rightclick.js

```diff
@@ -0,0 +1,23 @@
+document.onclick = hideMenu;

```

Comment:
Again, these are functions that we didn't talk about implementing but I'll merge them since they don't conflict with anything so we can revisit after we meet our current goals.